users/tests/test_permissions.py

```diff
@@ -0,0 +1,78 @@
+import pytest
+from django.contrib.auth.hashers import make_password
+
+from core.models import Community, CommunityMember, Post
+from users.models import User
+
+
+@pytest.fixture()
+def user() -> User:
```

Comment:
not needed:
https://github.com/pymasterspl/reddit/blob/6810df3c26114963e2be001cad26822025886179/conftest.py#L30